woocommerce/woocommerce-ios
# [Blaze] Use v1.1 endpoint to show campaign list

Part of: #12120


## Description

Use the v1.1 endpoint to fetch and display the Blaze campaign list and dashboard screens. 

In a future PR, we will remove the code for the previous Blaze campaign list endpoint. (`BlazeStore.synchronizeCampaigns`)

## Testing instructions

- Login into a Woo store with more than 30 Blaze campaigns. (The count is required to validate that pagination works)
- In the dashboard, you should see the latest campaign.
- Tap on the campaign to view the details in a web view
- Navigate back and tap on "Show All Campaigns"
- You will see the list of Blaze campaigns
- Scroll and validate that you can view all the Blaze campaigns.

## Screenshots


https://github.com/woocommerce/woocommerce-ios/assets/524475/5e703ae6-d3ff-42f5-a461-a4177dfccf02


---
